cmd/readOnlyRootFilesystem_test.go

```diff
@@ -2,12 +2,12 @@ package cmd
 
 import "testing"
 
-func TestSecurityContextNIL_RORF(t *testing.T) {
-	runAuditTest(t, "security_context_nil.yml", auditReadOnlyRootFS, []int{ErrorReadOnlyRootFilesystemNIL})
+func TestSecurityContextNil_RORF(t *testing.T) {
```

Comment:
Just a minor nit you can remove '_' as well since the camelCase is good enough :)